ieeeconf/setup.tex

```diff
@@ -0,0 +1,92 @@
+
+\ifx\mydocclass\undefined
+\def\mydocclass{conference}
+\fi
+
+\ifx\mycolumns\undefined
+\def\mycolumns{onecolumn}
+\fi
+
+\ifx\mysize\undefined
+\def\mysize{10pt}
+\fi
+
+\ifdefined\myinnermargin
+\newcommand{\CLASSINPUTinnersidemargin}{\myinnermargin}
+\fi
+\ifdefined\myoutermargin
+\newcommand{\CLASSINPUToutersidemargin}{\myoutermargin}
+\fi
+\ifdefined\mytopmargin
+\newcommand{\CLASSINPUTtoptextmargin}{\mytopmargin}
+\fi
+\ifdefined\mybottommargin
+\newcommand{\CLASSINPUTbottomtextmargin}{\mybottommargin}
+\fi
+
+\documentclass[\mysize,\mydocclass,\mycolumns]{ieeeconf}
```

Comment:
You are. There's a slight difference in the ordering of the footer, the `\overrideIEEEmargins` flag is added, and the author block is slightly different, but that's pretty much it.